src/server/api/index.js

```diff
@@ -1285,6 +1397,24 @@ function createAPI(app, mountPath, middlewareOpts) {
             });
     });
 
+    router.get('/projects/:ownerId/:projectName/commits/:commitHash/export', ensureAuthenticated,
+        function (req, res, next) {
+            console.log('arrived');
```

Comment:
Combine these two with `router.get(['/projects/:ownerId/:projectName/commits/:commitHash/export', '/projects/:ownerId/:projectName/commits/:commitHash/export*']).`
The first should also yield `req.params[0] === undefined`.